hack/agent_installer/start_db.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -euo pipefail
+set -o errexit
+
+su - postgres -c "mkdir -p /tmp/postgres/data"
+
+su - postgres -c "/usr/bin/initdb -D /tmp/postgres/data"
+su - postgres -c "/usr/bin/pg_ctl -D /tmp/postgres/data/ -l /tmp/postgres/logfile start"
```

Comment:
Ideally we'd want to do `exec postgres ...` so that the task runs in the foreground (and replaces the shell process). I notice that [in the upstream docker image](https://github.com/docker-library/postgres/blob/780680ebfa85d8220627985c0a16ecfd79d44a0f/10/bullseye/docker-entrypoint.sh#L323-L328) they temporarily start the server in the background using `pg_ctl`, create the user and DB, then stop the temporary server again and `exec postgres`.